OpenEdX/Managers/PushNotificationsManager/Listeners/FCMListener.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  FCMListener.swift
+//  OpenEdX
+//
+//  Created by Anton Yarmolenka on 24/01/2024.
+//
+
+import Foundation
+
+class FCMListener: PushNotificationsListener {
```

Comment:
I guess it will be implementing the `didReceiveRemoteNotification` as well.

Reply:
I added this function to the `PushNotificationsListener` protocol as an extension because it does the same thing for all listeners - checks this notification came from that listener, creates a `PushLink` from `userInfo` and calls the `processLinkFromNotification` function from `DeepLinkManager`.